vue/vue.config.js

```diff
@@ -1,3 +1,16 @@
+let RAPID_SKILLS_ENVIRONMENT = {};
+
+try {
+  RAPID_SKILLS_ENVIRONMENT = JSON.parse(process.env.RAPID_SKILLS_ENVIRONMENT);
```

Comment:
@littleforest here is what I am trying. Expecting it to be a JSON object string.